app/routes/questions/$question.tsx

```diff
@@ -102,7 +102,7 @@ export function Question({
   } else if (text == null) {
     html = 'Loading...'
   } else {
-    html = text
+    html = text;
```

Comment:
don't forget to run prettier with project settings please, not the defaults.

Reply:
Thanks!